sdk/nodejs/api/client.gen.ts

```diff
@@ -53,7 +53,16 @@ class BaseClient {
 }
 
 export type BuildArg = {
+  /**
+   * The build argument name.
+   * @example MODE
```

Comment:
I'll try to find a way to format it good!

Reply:
The problem with rolling back only for this case is that it creates inconsistency. I can easily handle that in the codegen but that will be inconsistent.
Instead, we shall maybe try to fix the theme to handle this case correctly. I'll see if it's possible.

Reply:
Ok I found a way! instead of giving an example in the field, we can give it in the exec call so it's appropriately displayed, and then we can remove the example for the field.
So only methods function calls will be documented with examples that make sense in a doc.

Reply:
Fixed in https://github.com/dagger/dagger/pull/4367/commits/2a0ebdce4d802f8666d5963ca8963641169fd915